src/components/site/SubPageTabItem.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+const SubPageTabItem = props => {
+  const { text, link, className } = props;
+  const TabItemClasses = classnames(className);
```

Comment:
local variables are usually lower case. I personally like the name `cssClasses` in this case. but you can probably come up with something better.

Reply:
This is in reference to `TabItemClasses`, correct? If so, would `itemClasses` work/ make sense?

Reply:
works for me, i like putting in css in the name, just because maybe you have something like vehicleClasses which relates to something different, but this is readable enough in my opinion.

Reply:
Sounds good. Updated and pushed up in the latest commit.